test/unit/past-purchase.ts

```diff
@@ -0,0 +1,33 @@
+import { Events, Selectors } from '@storefront/core';
+import * as sinon from 'sinon';
+import PastPurchase from '../../src/past-purchase';
+import suite from './_suite';
+
+const PAST = ['a', 'b'];
+
+suite('PastPurchase', ({ expect, spy, stub }) => {
+  let pastPurchase: PastPurchase;
+  let queryPastPurchases: sinon.SinonStub;
+
+  beforeEach(() => {
+    PastPurchase.prototype.select = <any>spy(() => PAST);
```

Comment:
if we modify the `prototype` here then we should be deleting the added key in an `afterEach()`